trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py

```diff
@@ -0,0 +1,135 @@
+        - dry engine weight
+        - fuel system weight
+        - thurst reversers weight
+        - electrical system weight
+        - starter engine weight
+        - nacelle weight
+        - cargo containers
+
+        Assumptions:
+            1) Rated thrust per scaled engine and rated thurst for baseline are the same
+            2) Engine weight scaling parameter is 1.15
+            3) Enginge inlet weight scaling exponent is 1
+            4) Baseline inlet weight is 0 lbs as in example files FLOPS
+            5) Baseline nozzle weight is 0 lbs as in example files FLOPS
+
+        Source:
+            The Flight Optimization System Weight Estimation Method
+
+        Inputs:
+            vehicle - data dictionary with vehicle properties                   [dimensionless]
+                -.design_mach_number: design mach number for cruise flight
+                -.mass_properties.max_zero_fuel: zero fuel weight               [kg]
+                -.systems.accessories: type of aircraft (short-range, commuter
+                                                        medium-range, long-range,
+                                                        sst, cargo)
+            prop - data dictionary with propulsion system properties
+                -.number_of_engines: number of engines
+                -.nacelle_diameter: diameter of nacelle                         [meters]
+                -.engine_length: length of complete engine assembly             [meters]
+                -.sealevel_static_thrust: thrust at sea level                   [N]
+
+
+        Outputs:
+            output - data dictionary with weights                               [kilograms]
+                    - output.wt_prop: total propulsive system weight
+                    - output.wt_thrust_reverser: thurst reverser weight
+                    - output.starter: starter engine weight
+                    - output.wt_engine_controls: engine controls weight
+                    - output.fuel_system: fuel system weight
+                    - output.nacelle: nacelle weight
+                    - output.wt_eng: dry engine weight
+
+        Properties Used:
+            N/A
+    """
+    NENG        = prop.number_of_engines
+    WNAC        = nacelle_FLOPS(prop)
+    WFSYS       = fuel_system_FLOPS(vehicle, NENG)
+    WENG        = engine_FLOPS(vehicle, prop)
+    WEC, WSTART = misc_engine_FLOPS(vehicle, prop)
+    WTHR        = thrust_reverser_FLOPS(prop)
+    WPRO        = NENG * WENG + WFSYS + WEC + WSTART + WTHR + WNAC
+
+    output                      = Data()
+    output.wt_prop              = WPRO
+    output.wt_thrust_reverser   = WTHR
+    output.wt_starter           = WSTART
+    output.wt_engine_controls   = WEC
+    output.fuel_system          = WFSYS
+    output.nacelle              = WNAC
+    output.wt_eng               = WENG * NENG
+    return output
+
+
+def nacelle_FLOPS(prop):
+    """ Calculates the weight of the nacelle for all engines"""
```

Comment:
These functions will all need docstrings with inputs and outputs

Reply:
FINC